connector/saml/saml.go

```diff
@@ -167,6 +186,14 @@ func (c *Config) openConnector(logger logrus.FieldLogger) (*provider, error) {
 		nameIDPolicyFormat: c.NameIDPolicyFormat,
 	}
 
+	var err error
+	if p.requestBinding, err = normalizeBinding(c.RequestBinding); err != nil {
+		return nil, err
+	}
+	if p.responseBinding, err = normalizeBinding(c.ResponseBinding); err != nil {
```

Comment:
the response binding should probably default to whatever the request binding is, right?

Reply:
not sure. In most cases, responseBinding will be POST. Even requestBinding is redirect. Would it be simpler to define fixed default values in config?

Reply:
> Would it be simpler to define fixed default values in config?

Works for me.